buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -752,11 +755,11 @@ public static int writeAscii(ByteBuf buf, CharSequence seq) {
     }
 
     // Fast-Path implementation
-    static int writeAscii(AbstractByteBuf buffer, int writerIndex, CharSequence seq, int len) {
+    static int writeAscii(AbstractByteBuf buffer, int writerIndex, CharSequence seq, int srcIndex, int len) {
 
         // We can use the _set methods as these not need to do any index checks and reference checks.
         // This is possible as we called ensureWritable(...) before.
-        for (int i = 0; i < len; i++) {
```

Comment:
Please verify that it won't prevent any vectorization optimization: see https://richardstartin.github.io/posts/4k-aliasing#dynamic-array-offsets-prevent-autovectorisation

Reply:
@timandy ^^

Reply:
@normanmaurer After perf test, rewrite a overload method is not necessary.